build.sbt

```diff
@@ -38,6 +39,29 @@ val betterMonadicForV = "0.3.1"
 lazy val scalaVersions =
   upickle.default.read[List[String]](new File("scalaVersions.json"))
 
+// format: off
+val coreDeps = Seq(
+  "org.typelevel"       %% "cats-core"                  % catsV,
+  "org.typelevel"       %% "cats-effect"                % catsEffectV,
+  "org.typelevel"       %% "cats-kernel"                % catsV,
+  "co.fs2"              %% "fs2-core"                   % fs2V,
+  "co.fs2"              %% "fs2-reactive-streams"       % fs2V,
+  "org.http4s"          %% "http4s-client"              % http4sV,
+  "org.http4s"          %% "http4s-core"                % http4sV,
+  "org.reactivestreams" %  "reactive-streams"           % reactiveStreamsV,
+  "org.scodec"          %% "scodec-bits"                % scodecV,
+  "io.chrisdavenport"   %% "vault"                      % vaultV,
+) ++ Seq(
+  "com.codecommit"      %% "cats-effect-testing-specs2" % catsEffectTestingV,
+  "org.http4s"          %% "http4s-blaze-server"        % http4sV,
+  "org.http4s"          %% "http4s-dsl"                 % http4sV,
+  "org.http4s"          %% "http4s-testing"             % http4sV,
+  "org.java-websocket"  %  "Java-WebSocket"             % javaWebsocketV,
+  "org.specs2"          %% "specs2-core"                % specs2V,
+  "org.specs2"          %% "specs2-scalacheck"          % specs2V,
+).map(_ % Test)
+// format: on
```

Comment:
Would be nice to tell scalafmt to align this section, but I think there are no local options.

Reply:
Hmm, I can't seem to get this to work. I now have 

```scala
// scalafmt: { align = most, trailingCommas = always }
val coreDeps = ???
// scalafmt: { align = none, trailingCommas = never }
```
and nothing changes...

But I think it is fine to merge this PR even without this extremely important formatting functionality :)

Reply:
We don't even have scalafmt on in the main repo, because it would be a pain across maintenance branches.  I'd like to resolve that someday, but again, we've lived without. :smile:

Reply:
Yeah, it is also not checked by scalafmtCheckAll, but it is certainly not very important..

I created scalameta/scalafmt#1861 for the formatting issue, but this PR should be ready now.